terraform/api.tf

```diff
@@ -34,16 +42,18 @@ module "api" {
     DJANGO_DANDI_DANDISETS_EMBARGO_LOG_BUCKET_NAME = module.sponsored_embargo_bucket.log_bucket_name
     DJANGO_DANDI_DOI_API_URL                       = "https://api.datacite.org/dois"
     DJANGO_DANDI_DOI_API_USER                      = "dartlib.dandi"
-    DJANGO_DANDI_DOI_API_PREFIX                    = "10.48324"
+    DJANGO_DANDI_DOI_API_PREFIX                    = "10.${var.dandi_instance_datacite_id}"
```

Comment:
Nitpick: It is reasonable to hardcode the `10.` since IIUC thats a universal for dois. However, IMO i think it will be friendlier to configure if the var would be set to `10.12345` because the full value is what is defined as "your prefix" from datacite. https://doi.test.datacite.org/repositories/dartlib.dandi/prefixes